disable -Werror and hardcoded optimization flag
gcc -> cc
remove unsupported flags -fweb and -frename-registers

Index: Quake/common.make
--- Quake/common.make.orig
+++ Quake/common.make
@@ -1,15 +1,15 @@
-CC = gcc
+CC = cc
 # HOST_CC is for bintoc which is run on the host OS, not
 #         the target OS: cross-compile friendliness.
-HOST_CC = gcc
+HOST_CC = cc
 LINKER = $(CC)
 STRIP ?= strip
 GLSLANG = glslangValidator
 DEBUG ?= 0
 
-CHECK_GCC = $(shell if echo | $(CC) $(1) -Werror -S -o /dev/null -xc - > /dev/null 2>&1; then echo "$(1)"; else echo "$(2)"; fi;)
+CHECK_GCC = $(shell if echo | $(CC) $(1) -S -o /dev/null -xc - > /dev/null 2>&1; then echo "$(1)"; else echo "$(2)"; fi;)
 
-CFLAGS += -MMD -Wall -Wno-trigraphs -Werror -std=gnu11
+CFLAGS += -MMD -Wall -Wno-trigraphs -std=gnu11
 CFLAGS += -D_FILE_OFFSET_BITS=64 
 CFLAGS += $(CPUFLAGS)
 ifneq ($(DEBUG),0)
@@ -18,9 +18,7 @@ CFLAGS += -g
 DO_STRIP=
 else
 DFLAGS += -DNDEBUG
-CFLAGS += -O3 -flto
-CFLAGS += $(call CHECK_GCC,-fweb,)
-CFLAGS += $(call CHECK_GCC,-frename-registers,)
+CFLAGS += -flto
 CFLAGS += $(call CHECK_GCC,-fno-asynchronous-unwind-tables,)
 CFLAGS += $(call CHECK_GCC,-fno-ident,)
 CMD_STRIP=$(STRIP) $(1)
